portfolio/src/main/webapp/comments/CommentFetcher.js

```diff
@@ -19,25 +19,38 @@ class CommentFetcher {
         return comments;
     }
 
+    async getBlobUploadUrl() {
+        const url = '/blobstore';
+
+        const res = await fetch(url);
+        const blobstoreUploadUrl = await res.text();
+
+        return blobstoreUploadUrl;
+    }
+
     /**
      * function that makes a POST request add a comment
      * @async
      * @param name the name of the author of the comment
```

Comment:
These should also have more detail with types in a structured fashion

@param {string} name the user name of the author of the comment 

etc